src/libsyntax/ext/build.rs

```diff
@@ -199,7 +199,7 @@ pub trait AstBuilder {
     fn item(&self, span: Span,
             name: Ident, attrs: Vec<ast::Attribute> , node: ast::ItemKind) -> P<ast::Item>;
 
-    fn arg(&self, span: Span, name: Ident, ty: P<ast::Ty>) -> ast::Arg;
+    fn arg(&self, span: Span, attrs: Vec<ast::Attribute>, name: Ident, ty: P<ast::Ty>) -> ast::Arg;
```

Comment:
Could you revert this change?
These APIs are kind of deprecated and it's better to leave them alone unless you actually want to pass non-empty attribute list to `.arg(...)` (which you don't as far as I can see).